src/Installers/Windows/SharedFrameworkBundle/SharedFrameworkBundle.wixproj

```diff
@@ -65,8 +67,15 @@
     <PackageFileName>$(RuntimeInstallerBaseName)-$(PackageVersion)-win-$(Platform)$(TargetExt)</PackageFileName>
 
     <BundleNameShort>Microsoft ASP.NET Core $(PackageBrandingVersion)</BundleNameShort>
+    <!--
+      Title still truncated to e.g. "Microsoft ASP.NET Core 8.0.0 Release".
+      ??? Thoughts on sharing branding w/ the hosting bundle e.g. "Microsoft .NET 8.0.0 Release Candidate 1"?
+      ??? If we go that route, could make the subtitle "ASP.NET Shared Framework" or "ASP.NET Core...".
+    -->
+    <BundleNameShortNoBuild>Microsoft ASP.NET Core $(PackageBrandingVersionNoBuild)</BundleNameShortNoBuild>
```

Comment:
I think this is fine how it is, if anything I'd change the hosting bundle title since it isn't exactly "Microsoft .Net", I think that title would apply more appropriately to the SDK. But I don't think it's critical

Reply:
/ping @joeloff